package.json

```diff
@@ -142,7 +142,7 @@
     "test-client:watch": "nodemon -e js,jsx --exec npm run test-client",
     "test-server:watch": "nodemon -e js,jsx --exec npm run test-server",
     "test-test:watch": "nodemon -e js,jsx --exec npm run test-test",
-    "lint": "bin/run-lint",
+    "lint": "bin/run-lint .",
```

Comment:
Why did you change the way we call `run-lint`? I am not saying it’s wrong, just wondering why.